graphql-database-manager-dynamo/src/main/java/com/fleetpin/graphql/database/manager/dynamo/DynamoDbManager.java

```diff
@@ -110,7 +116,7 @@ public DynamoDbManager build() {
 				idGenerator = () -> UUID.randomUUID().toString();
 			}
 
-			database = Objects.requireNonNullElse(database, new DynamoDb(mapper, tables, client, idGenerator));
+			database = Objects.requireNonNullElse(database, new DynamoDb(mapper, tables, historyTable, client, idGenerator));
```

Comment:
historyTable is optional.
Would not have this here.
If someone tried to perform a query though and no history table should throw a nicer error than nullpointer